types/ws/index.d.ts

```diff
@@ -87,18 +93,22 @@ declare namespace WebSocket {
 
     export interface IClientOptions {
         protocol?: string;
-        agent?: http.Agent;
+        perMessageDeflate?: boolean | IPerMessageDeflateOptions;
+        localAddress?: string;
+        protocolVersion?: number;
         headers?: { [key: string]: string };
-        protocolVersion?: any;
-        host?: string;
         origin?: string;
-        pfx?: any;
-        key?: any;
+        agent?: http.Agent;
+        host?: string;
+        family?: number;
+        checkServerIdentity?: Function;
+        rejectUnauthorized?: boolean;
         passphrase?: string;
-        cert?: any;
-        ca?: any[];
         ciphers?: string;
-        rejectUnauthorized?: boolean;
+        cert?: any;
```

Comment:
This is also `string|string[]|Buffer|Buffer[]` also I _think_ string should in lowercase? In that case it should be fixed below.